modules/apps/users-admin/users-admin-web/src/main/java/com/liferay/users/admin/web/internal/servlet/taglib/util/UserActionDropdownItems.java

```diff
@@ -107,6 +108,8 @@ public List<DropdownItem> getActionDropdownItems() {
 						PropsValues.PORTAL_IMPERSONATION_ENABLE &&
 						(_user.getUserId() != _themeDisplay.getUserId()) &&
 						_user.isActive() && !_themeDisplay.isImpersonated() &&
+						!Objects.equals(
```

Comment:
Can you move this after PropsValues.PORTAL_IMPERSONATION_ENABLE but before (_user.getUserId() != _themeDisplay.getUserId()) ? Thx